hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -320,6 +321,45 @@ public boolean addSCM(AddSCMRequest request) throws IOException {
     }
   }
 
+  @Override
+  public boolean removeSCM(RemoveSCMRequest request) throws IOException {
+    final List<RaftPeer> newRaftPeerList =
+        new ArrayList<>(getDivision().getGroup().getPeers());
```

Comment:
instead of calling `getDivision()` here can we not just use the `division` variable declared in the `SCMRatisServicerImpl`?